tests/scripts/minikube.sh

```diff
@@ -38,17 +31,7 @@ wait_for_ssh() {
 }
 
 copy_image_to_cluster() {
-    local build_image=$1
-    local final_image=$2
-    local helm_image_tag=
-
-    echo copying ${build_image} to minikube
-    mkdir -p ${WORK_DIR}/tests
-    docker save -o ${tarfile} ${build_image}
-    minikube_scp ${tarfile} /home/docker
-    minikube_ssh "docker load -i /home/docker/${tarname}"
-    minikube_ssh "docker tag ${build_image} ${final_image}"
-
+    docker save $1 | (eval $(minikube docker-env) && docker load && docker tag $1 $2)
```

Comment:
would be nice to keep the local vars for readability instead of using `$1` and `$2` directly

Reply:
sounds good